src/components/personalSettings/personalSettings.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+
+import style from './personalSettings.module.css';
+import { setCurentPathNameThunkCreator } from '../../redux/navbar-reducer';
+import ProfileSettings from './profileSettings/profileSettings';
+import AccountSetting from './accountSettings/accountSettings';
+import SecuritySettings from './securitySettings/securitySettings';
+
+function PersonalSettings(props) {
+  const subSettings = props.match.params.subsettings;
+  props.setCurentPathNameThunkCreator('');
```

Comment:
always leave 1 empty line before return
```suggestion
  props.setCurentPathNameThunkCreator('');
  
```